lighthouse-core/test/gather/gather-runner-test.js

```diff
@@ -1114,26 +1161,30 @@ describe('GatherRunner', function() {
       navigationError = /** @type {LH.LighthouseError} */ (new Error('NAVIGATION_ERROR'));
     });
 
-    it('passes when the page is loaded', () => {
+    it('passes when the page is loaded and doc type is text/html', () => {
```

Comment:
for these tests, I think you can 

1) leave the test name the same
2) prefer `mainRecord.mimeType = 'text/html'` over defining a variable

Reply:
Thanks, fixed both of these